src/boot/efi/boot.c

```diff
@@ -2645,6 +2659,32 @@ static void config_load_all_entries(
                 config_add_entry(config, entry);
         }
 
+        // XXX: nuke the o/b from the help screen ?
+        // XXX: add FLAGS_SET(get_os_indications_supported(), EFI_OS_INDICATIONS_xx)) ?
```

Comment:
hmm, what is this about? shutdown doesn't show up in OS indications, so what is this comment about?